make-boot-image/make-boot-image-from-kernel

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env bash
```

Comment:
Does this actually require bash, or is it just a convenience?

Reply:
We make use of the `if [[ condition ]]` construct which, whilst supported by bash / busybox sh, is notably not supported by dash.

We also make use of arrays for TEMP_DIRS which definitely feels like a bashism.